smartcontract/service/native/governance/method.go

```diff
@@ -1330,7 +1332,7 @@ func executeCommitDpos2(native *native.NativeService, contract common.Address) e
 	return nil
 }
 
-func splitNodeFee(native *native.NativeService, contract common.Address, peerPubkey string, peerAddress common.Address, ifConsensus bool, totalPos uint64, nodeAmount uint64) error {
+func splitNodeFee(native *native.NativeService, contract common.Address, peerPubkey string, peerAddress common.Address, preIfConsensus, ifConsensus bool, totalPos uint64, nodeAmount uint64) error {
```

Comment:
Should it not be `preIfConsensus bool`?

Reply:
it shares same type with its next var `ifConsensus`.

Reply:
Sounds good